src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,35 @@
 using namespace std;
 
 int t06_cinema() {
+    int N=0, M=0, x=0, flag=16, tmp=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'tmp' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734892)